codec-ntg/README.md

```diff
@@ -0,0 +1,13 @@
+# NTG codec
+
+This codec can be used for decoding and encoding message in NTG (Native) protocol.
```

Comment:
This codec can be used for decoding and encoding **messages** in NTG (Native) protocol.